package.json

```diff
@@ -1,5 +1,32 @@
 {
+  "name": "istanbuljs",
+  "version": "1.0.0",
```

Comment:
But this is just an internal number right? Are external deps requiring `istanbuljs` right now? Unless I'm misunderstanding a lerna concept right now...